packages/frameworks/frameworks.json

```diff
@@ -229,6 +233,7 @@
     "demo": "https://scully.now-examples.now.sh",
     "tagline": "Scully is a static site generator for Angular.",
     "description": "The Static Site Generator for Angular apps.",
+    "website": "https://github.com/scullyio/scully",
```

Comment:
Scully didn't have a website - their [GitHub repo](https://github.com/scullyio/scully) acts as documentation, so I used that.